sdv/multi_table/utils.py

```diff
@@ -0,0 +1,431 @@
+"""Utility functions for the MultiTable models."""
+import math
+from collections import defaultdict
+from copy import deepcopy
+
+import numpy as np
+import pandas as pd
+
+from sdv._utils import _get_root_tables
+from sdv.multi_table import HMASynthesizer
+from sdv.multi_table.hma import MAX_NUMBER_OF_COLUMNS
+
+MODELABLE_SDTYPE = ['categorical', 'numerical', 'datetime', 'boolean']
+
+
+def _get_relationship_for_child(relationships, child_table):
+    return [rel for rel in relationships if rel['child_table_name'] == child_table]
+
+
+def _get_relationship_for_parent(relationships, parent_table):
```

Comment:
minor: can we make plural (ie. get_relationships)

Reply:
Yes, done in [209b720](https://github.com/sdv-dev/SDV/pull/1874/commits/209b720e703f9fef6eb4ceb112c5506804198b66)